ui-tests/cypress/integration/pages/management.update.spec.js

```diff
@@ -0,0 +1,137 @@
+import HomePage from "../../support/pageObjects/HomePage";
+import { generateRandomId } from '../../support/utils';
+
+let currentPage;
+
+const openManagementPage = () => {
+    cy.visit('/');
+    let currentPage = new HomePage();
+    currentPage = currentPage.getMenu().getPages().open();
+    return currentPage.openManagement();
+}
+
+const OOTB_PAGE_TEMPLATES = ['1-2-column', '1-2x2-1-column', '1-2x4-1-column', '1-column', 'content-page', 'home', 'single_frame_page']
+const OOTB_OWNER_GROUPS = ['Administrators', 'Free Access'];
+const OOTB_CODE = 'sitemap';
+
+describe('Pages Management - Update', () => {
+    beforeEach(() => {
+        cy.kcLogin("admin").as("tokens");
+    });
+
+    afterEach(() => {
+        cy.kcLogout();
+    });
+
+  it('reuse existing code page', () => {
+    currentPage = openManagementPage();
+
+    // go to add page
+    currentPage = currentPage.getContent().clickAddButton();
+
+    currentPage.getContent()
+        .fillRequiredData(`Existing code`, `Existing code`, OOTB_CODE, 0, OOTB_PAGE_TEMPLATES[0]);
+
+    currentPage.getContent().clickSaveButton();
+
+    cy.wait(2000);
+    // should not be able to save page with existing page code
+    currentPage.getContent().getAlertMessage().contains(OOTB_CODE).should('be.visible');
+  });
+
+  it('use non existing page template', () => {
+    currentPage = openManagementPage();
+
+    // go to add page
+    currentPage = currentPage.getContent().clickAddButton();
+
+    currentPage.getContent().getPageTemplateSelector().children('option').then(options => {
+      [...options].forEach(o => {
+
+        if (o.value) {
+          // should contain all OOTB values
+          expect(OOTB_PAGE_TEMPLATES.includes(o.value)).to.eq(true);
```

Comment:
It would be easier to read if you were expecting the list to include the element ...
i.e. `expect(OOTB_PAGE_TEMPLATES).to.include(o.value)`